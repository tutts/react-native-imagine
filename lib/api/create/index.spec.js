const create = require('./')

describe('Create', () => {
  it('creates a map with symlinks', () => {
    expect({}).toMatchSnapshot()
  })
})
