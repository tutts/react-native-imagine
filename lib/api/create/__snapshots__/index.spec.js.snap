// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create creates a map with symlinks 1`] = `Object {}`;
